core/src/test/java/org/apache/calcite/test/SqlValidatorTest.java

```diff
@@ -10093,22 +10093,22 @@ private void checkCustomColumnResolving(String table) {
         + "from orders\n"
         + "group by floor(rowtime to hour)")
         .fails("Call to auxiliary group function 'TUMBLE_END' must have "
-            + "matching call to group function 'TUMBLE' in GROUP BY clause");
+            + "matching call to group function '\\$TUMBLE' in GROUP BY clause");
```

Comment:
Do we need this change ? 
It seems we are recommending to write the sql as `GROUP BY $TUMBLE(rowtime, interval '2' hour, time '00:12:00')`

Reply:
HI @jinxing64. #1481 added TUMBLE to parser so the syntax is still remain the same. E.g. `GROUP BY TUMBLE(rowtime, interval '2' hour, time '00:12:00')`. 

The decision in #1481 was still return `TUMBLE` as the operator name. And in order to work on https://jira.apache.org/jira/browse/CALCITE-3340, it turns out that we should return `$TUMBLE` as the operator name (at least it is a more straightforward way).  `$TUMBLE` will only appear when you call Operator.`getName()` or `toString()`.

Reply:
Thanks for explain ~
My concern is that the failing message seems confusing to recommend to write the sql as `GROUP BY $TUMBLE(rowtime, interval '2' hour, time '00:12:00')`

Reply:
Yes I think it's a valid concern. Ideally we should start to deprecate GROUP BY TUMBLE as table function TUMBLE is implemented. Maybe we can tolerate a bit for now on this concern? Or we should have a pending table function TUMBLE implementation to justify this PR and then merge both one by one? 

I would be open to suggestions.